dev-requirements.txt

```diff
@@ -6,14 +6,13 @@ pytest-cov
 Mock
 
 #Linting
+enum34
```

Comment:
Errors occurred on the Jenkins build for these to packages related to `flake8` linting when using a clean virtualenv. It could be that something was missing in its dependencies list etc. Adding these resolved the problem on Jenkins.